src/components/DryRunnerCardLog/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Column from '../Column'
+import Image from '../Image'
+import Row from '../Row'
+import arrayPad from '../../helpers/arrayPad'
+import './index.css'
+
+const DryRunnerCardLog = props => {
+  return (
+    <div className='DryRunnerCardLog'>
+      <Row>
+        {arrayPad(props.cards.slice(0, 6), 6, null, +1).map((card, index) => (
```

Comment:
Maybe we put that in a `cards` constant before the return statement for clarity?